### PR TITLE
clean cmd copy

### PR DESCRIPTION
**Pull request by tomermevorach**

### CODE REVIEW

**Review by netama — COMMENTED** [review]
LGTM (leaving it up to Gil to approve).
We also talked about maybe clarifying that `--no` can be prefixed to any of flags to disable - might be worth adding too? not sure what's the best place for it (the command description or one/all of the flags), because it's a generic cli feature

**Comment by tomermevorach** [comment]
> LGTM (leaving it up to Gil to approve).
> We also talked about maybe clarifying that `--no` can be prefixed to any of flags to disable - might be worth adding too? not sure what's the best place for it (the command description or one/all of the flags), because it's a generic cli feature

Thats a general CLI UX issue, not related just to this specific command right?

**Comment by netama** [comment]
> > LGTM (leaving it up to Gil to approve).
> > We also talked about maybe clarifying that `--no` can be prefixed to any of flags to disable - might be worth adding too? not sure what's the best place for it (the command description or one/all of the flags), because it's a generic cli feature
> 
> Thats a general CLI UX issue, not related just to this specific command right?

Yes

File: packages/cli/src/commands/clean.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ const diffBuilder = createCommandBuilder({
   options: {
     command: 'clean',
     // marking description as false will make this a hidden command (only accessible directly)
```
Comment (adamro) [inline packages/cli/src/commands/clean.ts:89]: The comment in line 89 is redundant now that we are adding a description

**Review by gilhoffer — APPROVED** [review]
LGTM

File: packages/cli/src/commands/clean.ts (hunk 1 above)
Comment (tomermevorach) [reply packages/cli/src/commands/clean.ts]: thx removed the comment

**Pull request merged**